### PR TITLE
CRUD funtions and small change migrations

### PR DESCRIPTION
**Pull request by MatzLeeflang**
made the CRUD functionalities for the controllers.
and made a small change to the migrations so when u delete a parent row it doesnt give an error and deletes al the children of the parent.

### CODE REVIEW

**Review by streikbaut — APPROVED** [review]
Looks good, same question as before. Does it matter that not all heat pumps have every kind of data, while they are all required in the code? If this doesn't matter then the rest looks fine and you can merge it.

**Comment by MatzLeeflang** [comment]
> Looks good, same question as before. Does it matter that not all heat pumps have every kind of data, while they are all required in the code? If this doesn't matter then the rest looks fine and you can merge it.

fixed it now

File: api/app/Http/Controllers/SupplierController.php [hunk 1]
```diff
@@ -4,6 +4,9 @@
 
 use App\Helpers\Response;
 use App\Models\Supplier;
+use App\Models\Type;
+use App\Models\boundary;
```
Comment (streikbaut) [inline api/app/Http/Controllers/SupplierController.php:8]: Ik weet niet of dit voor problemen zorgt maar boundary is hier zonder hoofdletter terwijl Boundary.php in Models wel met hoofdletter is.

**Review by streikbaut — APPROVED** [review]
Looks good

**Pull request merged**